litellm/proxy/proxy_server.py

```diff
@@ -455,57 +464,49 @@ def model_list():
     else:
         all_models = litellm.utils.get_valid_models()
         return dict(
-            data = [{"id": model, "object": "model", "created": 1677610602, "owned_by": "openai"} for model in all_models],
+            data=[{"id": model, "object": "model", "created": 1677610602, "owned_by": "openai"} for model in
+                  all_models],
             object="list",
         )
 
+
+@router.post("/v1/completions")
 @router.post("/completions")
```

Comment:
The logic of the two routes is similar, one of which gives default parameters for litellm_completion and the other does not. It's confusing. I've combined them into one function here